contracts/StorageContract.sol

```diff
@@ -166,6 +166,27 @@ abstract contract StorageContract is DecentralizedKV {
         emit MinedBlock(shardId, diff, info.blockMined, minedTs, miner, minerReward);
     }
 
+    function miningReward(uint256 shardId, uint256 blockNumber) public view returns (uint256) {
+        uint256 minedTs = block.timestamp - (block.number - blockNumber) * 12;
+
+        MiningLib.MiningInfo storage info = infos[shardId];
```

Comment:
can this be reused by _rewardMiner?

Reply:
prepaidLastMineTime is needed to be updated in _rewardMiner which makes it is not easy to be reused